src/styles/card.scss

```
@@ -0,0 +1,97 @@
+.card {
+  width: 200px;
+  padding: 0 16px;
+  margin: 0 24px 46px;
+  border: 1px solid #f3f3f3;
```

You created variables for colors, but not for all 🥺 
```suggestion
  border: 1px solid $c-color;
```